resources/definitions/winbo_superhelper105.def.json

```diff
@@ -0,0 +1,127 @@
+{
+    "version": 2,
+    "name": "Winbo Super Helper 105",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "author": "Winbo",
+        "manufacturer": "Winbo Smart Tech Co., Ltd.",
+        "category": "Other",
+        "visible": true,
+        "file_formats": "text/x-gcode",
+        "supports_usb_connection": true,
+        "machine_extruder_trains":
+        {
+            "0": "winbo_superhelper105_extruder"
+        }
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "Winbo Super Helper 105" },
+        "machine_width": { "default_value": 108 },
+        "machine_depth": { "default_value": 108 },
+        "machine_height": { "default_value": 158 },
+        "machine_heated_bed": { "default_value": false },
+        "material_bed_temp_wait": { "default_value": false },
+        "machine_filament_park_distance": { "value": "machine_heat_zone_length" },
+        "machine_nozzle_heat_up_speed": { "default_value": 1.4 },
+        "machine_nozzle_cool_down_speed": { "default_value": 0.8 },
+        "machine_head_with_fans_polygon":
+        {
+            "default_value":
+            [
+                [ -110, 50 ],[ -110, -30 ],[ 110, -30 ],[ 110, 50 ]
+            ]
+        },
+        "machine_gcode_flavor": { "default_value": "Marlin" },
+        "machine_max_feedrate_x": { "default_value": 250 },
+        "machine_max_feedrate_y": { "default_value": 200 },
+        "machine_max_feedrate_z": { "default_value": 40 },
+        "machine_acceleration": { "default_value": 2000 },
+        "gantry_height": { "default_value": 200 },
+        "machine_extruder_count": { "default_value": 1 },
+        "machine_start_gcode": { "default_value": "G21\nG90\nM82\nM107\nG28 X0 Y0\nG28 Z0\nG1 F6000 Z0.3\nG92 E0\nG1 F1000 X30 E8\nG92 E0\nM117 Printing." },
+        "machine_end_gcode": { "default_value": "M104 S0\nM140 S0\nG92 E2\nG1 E0 F200\nG28 X0 Y0\nM84 X Y E" },
+        "material_diameter": { "default_value": 1.75 },
```

Comment:
As I already commented on PR #4102, you can remove the settings `material_diameter` and `machine_nozzle_size` from here and define them in the extruder definition file.